ios/brave-ios/Tests/BraveWalletTests/PortfolioStoreTests.swift

```diff
@@ -397,13 +396,62 @@ import XCTest
         ),
       ].filter { networkAsset in networks.contains(where: { $0 == networkAsset.network }) }
     }
+
+    let btcBalanceInSatoshi =
+      formatter.weiString(
+        from: mockBTCBalanceAccount1,
+        radix: .decimal,
+        decimals: Int(BraveWallet.BlockchainToken.mockBTCToken.decimals)
+      ) ?? ""
+    let btcTestnetBalanceInSatoshi =
+      formatter.weiString(
+        from: mockBTCBalanceTestnet,
+        radix: .decimal,
+        decimals: Int(BraveWallet.BlockchainToken.mockBTCToken.decimals)
+      ) ?? ""
+    let bitcoinWalletService = BraveWallet.TestBitcoinWalletService()
+    bitcoinWalletService._balance = { accountId, completion in
+      if accountId.uniqueKey == self.btcAccount1.accountId.uniqueKey {
+        completion(
+          .init(
+            totalBalance: UInt64(btcBalanceInSatoshi) ?? 0,
+            availableBalance: UInt64(btcBalanceInSatoshi) ?? 0,
+            pendingBalance: 0,
+            balances: [:]
+          ),
+          nil
+        )
+      } else if accountId.uniqueKey == self.btcAccount2.accountId.uniqueKey {
+        completion(
+          .init(
+            totalBalance: 0,
+            availableBalance: 0,
+            pendingBalance: 0,
+            balances: [:]
+          ),
+          nil
+        )
+      } else {
+        completion(
+          .init(
+            totalBalance: UInt64(btcTestnetBalanceInSatoshi) ?? 0,
+            availableBalance: UInt64(btcTestnetBalanceInSatoshi) ?? 0,
+            pendingBalance: 0,
+            balances: [:]
+          ),
+          nil
+        )
+      }
```

Comment:
I think these else statements be flipped? So we return 0 for all unknown BTC accounts, instead of specifically returning 0 for `btcAccount2`
```suggestion
      } else if accountId.uniqueKey == self.btcTestnetAccount.accountId.uniqueKey {
        completion(
          .init(
            totalBalance: UInt64(btcTestnetBalanceInSatoshi) ?? 0,
            availableBalance: UInt64(btcTestnetBalanceInSatoshi) ?? 0,
            pendingBalance: 0,
            balances: [:]
          ),
          nil
        )
      } else {
        completion(
          .init(
            totalBalance: 0,
            availableBalance: 0,
            pendingBalance: 0,
            balances: [:]
          ),
          nil
        )
      }
```

Reply:
there are intended 3 bitcoin accounts, two on mainnet and one on testnet. 
one bitcoin account on mainnet has balance and the other one has 0 to test hiding smaller balance. 
one bitcoin testnet account has balance to test bitcoin balance fetching correctly along with btc's network and account's keyringId. this testnet account also used to tested the sorting order (checking total fiat balance first then checking network is primary or testnet)

Reply:
Yeah I just meant if 2 accounts (1 on mainnet, 1 on testnet have a non-zero balance, shouldn't those 2 accounts be the ones explicitly defined in the test? More of a nit, unlikely to add another bitcoin account in the tests here.